src/main/scala/com.snowplowanalytics/weather/providers/darksky/DarkSky.scala

```diff
@@ -0,0 +1,22 @@
+package com.snowplowanalytics.weather
+package providers.darksky
+
+import cats.effect.Sync
+
+object DarkSky {
+
+  private[darksky] def basicClient[F[_]: Sync](transport: Transport[F]): DarkSkyClient[F] =
+    new DarkSkyClient(transport)
+
+  /**
+    * Create `DarkSkyClient` with underlying `HttpTransport` instance
+    * @param appId API key from Dark Sky
+    * @param apiHost URL to the Dark Sky API endpoints
+    * @param ssl whether to use https
+    */
+  def basicClient[F[_]: Sync](appId: String,
+                              apiHost: String = "api.darksky.net/forecast",
+                              ssl: Boolean    = true): DarkSkyClient[F] =
```

Comment:
do they also have both a http and https api? :thinking:

Reply:
Whoops, missed that - only https